LastWork(newWork)/Views/evaluations/evaluationsPage.js

```diff
@@ -0,0 +1,144 @@
+const EvaluationsTableHeader = {
+    init(option) {
+        const headings = [];
+        option.headings.forEach((el) => {
```

Comment:
There are more appropriate ways of mapping a collection's items to a particular piece of information than iterating through the collection via `forEach` and then `join`ing the results. Do you know what those ways might be?